echopype/echodata/simrad.py

```diff
@@ -0,0 +1,233 @@
+"""
+Contains functions that are specific to Simrad echo sounders
+"""
+from typing import Optional, Tuple
+
+from .echodata import EchoData
+
+
+def _check_mode_input_without_data(
+    waveform_mode: str, encode_mode: str, pulse_compression: bool
+) -> None:
+    """
+    Checks that the ``waveform_mode`` and ``encode_mode`` have
+    the correct values and that the pair of inputs are valid, without
+    considering the actual data.
```

Comment:
since there are 3 params here, so not just a pair
```suggestion
    Checks that the ``waveform_mode`` and ``encode_mode`` have
    the correct values and that the combination of input arguments are valid, without
    considering the actual data.
```